### DISCUSSION TITLE
Does anything support CELLNET Itron C1SC FCC SK9C1A?

### DISCUSSION THREAD

**Discussion by Whitestone27** [bemasher/rtlamr#291] [Q&A]
Looking for positive confirmation that this project cannot support my meter, and if anyone knows of alternate solutions.  

I also have a gas meter "Cellnet residential gas module" (FCC ID R7PEG0R3S2 ) and both devices have a "PowerLAN" address.  According to the FCC ID, the RF module will transmit a single channel of data at the 917 MHz using a Binary Phase Shift Keying (BPSK) spread spectrum transmitter.

Thank you!

**Comment by telxonius** [comment]
I have one of these Itron C1SC cellnet electric meters I bought on Ebay, and I have not been able to get this program or the one included on my Hackrf to read it. i believe it's a proprietary protocol that's not well documented publicly.
